index.html

```diff
@@ -19,120 +20,182 @@
                         <span class="bar"></span>
                         <span class="bar"></span>
                     </div>
-                </nav>
-            </header>
-        </div>
-        <div class="section">
-            <section>
-                <h1>Rudy<br>Menekam</h1>
-                <p> Hello! I am a software developer and designer! I can help you build a product, feature or website. Taka a look at my works. If you like what you see and have a project you need coded or designed, don’t hesitate and contact me.</p>
-                <a href="" class="button">Start collaboration</a>
-            </section>
+                    <di class="menu_bar">
+                        <ul>
+                            <li><a>Hello</a></li>
+                            <li><a>Portfolio</a></li>
+                            <li><a>About</a></li>
+                            <li><a>Contact</a></li>
+                        </ul>
```

Comment:
[Optional] Kindly ensure that the links in the navigation lead to the right sections of the website. Also, for semantic purposes, it'll be great to use a `nav` tag rather than the generic `div` tag.